blockchain/message_pool/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+pub mod block_prob;
```

Comment:
does this need to be public? Assume you are doing this to avoid linting dead code, so maybe add a TODO or something to remove public scope after used? Or just ignore the dead code smell?

Reply:
Good call, Im adding the TODO